src/07_slices.py

```diff
@@ -12,26 +12,32 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+slice1 = slice(1,2)
+print(a[slice1])
```

Comment:
Nice! In Python, we can access indexes just like we do in JS. Here, `print(a[1])` would have also worked